libdrgn/python/type.c

```diff
@@ -1578,13 +1586,16 @@ static DrgnType *compound_type(PyObject *tag_obj, PyObject *size_obj,
 		if (!cached_members_obj)
 			return NULL;
 		num_members = PyTuple_GET_SIZE(cached_members_obj);
+		members = malloc(sizeof(struct drgn_type_member) * num_members);
```

Comment:
This malloc is in a bit of a weird place, it is created and built up here, but it is actually used below. Since the condition is the same as here, I think it should be safe but I was wondering if there was a better/cleaner way to do this.

Reply:
I'm okay with having it here. However, please use `malloc_array(num_members, sizeof(*members))`.